docs/platforms/apple/common/enriching-events/viewhierarchy/index.mdx

```diff
@@ -14,6 +14,10 @@ Sentry makes it possible to render a JSON representation of the view hierarchy o
 
 This feature only applies to SDKs with a user interface, such as the ones for mobile and desktop applications. In some environments like native iOS, rendering the view hierarchy requires the UI thread and in the event of a crash, that might not be available. Another example where the view hierarchy might not be available is when the event happens before the screen starts to load. So inherently, this feature is a best effort solution.
 
+<Note>
+App hang events will not have view hierarchy because the main thread is blocked and we can't interact with UI elements in a background view.
```

Comment:
```suggestion
App hang events will not have view hierarchy because the main thread is blocked and Sentry can't interact with UI elements in a background view.
```